ds-css/src/styles/custom/utilities/mixins.scss

```diff
@@ -1 +1,25 @@
-// no empty source file allowed fix
\ No newline at end of file
+// Responsiveness
+@mixin tablet-portrait-view {
+  @media only screen and (min-width: 600px) {
```

Comment:
the mixins have different formatting (space before after @content)

Reply:
I made the change to line 4. Will push update